web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -225,6 +225,23 @@
             var type,
               layer = layers[i];
             if (layer.group == "Background layers") {
+              var layerAttributionArray = undefined;
```

Comment:
Nitpick: 

If you declare `layerAttributionArray`just with `var layerAttributionArray` it is by default `undefined`. No need to define it with `undefined`.